libs/json-glib/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2020 Sebastian Kemper <sebastian_ml@gmx.net>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=json-glib
+PKG_VERSION:=1.4.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://download.gnome.org/sources/json-glib/1.4
+PKG_HASH:=720c5f4379513dc11fd97dc75336eb0c0d3338c53128044d9fabec4374f4bc47
+
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+PKG_LICENSE:=LGPL-2.1+
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Sebastian Kemper <sebastian_ml@gmx.net>
+
+PKG_BUILD_DEPENDS:=gettext glib2/host meson/host
+
+PKG_FORTIFY_SOURCE:=0
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+include ../../devel/meson/meson.mk
+
+define Package/json-glib
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=JSON GLib Library
+  URL:=https://wiki.gnome.org/Projects/JsonGlib
+  DEPENDS:=+glib2
+endef
+
+define Package/json-glib/description
+JSON-GLib is a library providing serialization and deserialization
+support for the JavaScript Object Notation (JSON) format described by
+RFC 4627.
+endef
+
+MESON_ARGS += \
+	-Ddocs=false \
+	-Dintrospection=false \
+	-Dman=false
+
+# Disable installed-tests; this also indirectly removes
```

Comment:
Interestingly, upstream added tests as an option. No idea if it makes sense to backport.

Reply:
It's already disabled with sed script.